code/form/LocatorForm.php

```diff
@@ -30,11 +44,19 @@ public function __construct(Controller $controller, $name)
         if ($categories) {
             $categoriesField = DropdownField::create('CategoryID')
                 ->setTitle('')
-                ->setEmptyString('All Categories')
+                ->setEmptyString('all categories')
                 ->setSource($categories->map());
             $fields->push($categoriesField);
         }
 
+        if (Config::inst()->get('LocatorForm', 'show_radius')) {
+            $radiusArray = Config::inst()->get('LocatorForm', 'radius_array');
```

Comment:
This could hold true for line 52 as well, but testing should confirm this is the case.